test/tests/u-options.json

```diff
@@ -102,13 +89,7 @@
       "expParts": [
         { "type": "text", "value": "hello " },
         { "type": "bidiIsolation", "value": "\u2066" },
-        {
-          "type": "string",
-          "source": "|world|",
-          "dir": "ltr",
-          "id": "foo",
-          "value": "world"
-        },
+        { "type": "string", "dir": "ltr", "id": "foo", "value": "world" },
```

Comment:
any reason to lose the formatting?

Reply:
Do you mean that this is on a single line, rather than the multiple lines it had previously? It's because with the change, it's now short enough to easily fit on a single line.